contracts/src/messaging/IMerkleTreeMessageBus.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./Structs.sol";
+
+interface IMerkleTreeMessageBus {
+    // This function is called to add a cross chain state root to the message bus.
+    function addStateRoot(bytes32 stateRoot, uint256 activationTime) external;
+    // This function disables a cross chain state root from the message bus. On challenge
+    function blockStateRoot(bytes32 stateRoot) external;
```

Comment:
maybe a different name for this.
"block" - makes you think of the state root of a block.